packages.yml

```diff
@@ -0,0 +1,6 @@
+packages:
+  - package: fivetran/fivetran_utils
+    version: [">=0.4.0", "<0.5.0"]
+
+  - package: dbt-labs/spark_utils
+    version: [">=0.3.0", "<0.4.0"]
```

Comment:
Additionally, do we require spark utils as well? Could we simplify this down to just a dbt_utils dependency?